documentation/modules/auxiliary/scanner/ip/ipidseq.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+
+This ```auxiliary/scanner/ip/ipidseq``` module will probe hosts' IPID sequences and classify them using the same method Nmap uses when it's performing its IPID Idle Scan (-sI) and OS Detection (-O).
+
+More information: https://nmap.org/book/idlescan.html
+
+Nmap's probes are SYN/ACKs while this module's are SYNs.
+While this does not change the underlying functionality, it does change the chance of whether or not the probe will be stopped by a firewall.
+
+Nmap's Idle Scan can use hosts whose IPID sequences are classified as "Incremental" or "Broken little-endian incremental".
+
+## Verification Steps
+
+1. Start msfconsole
+1. Do: `use auxiliary/scanner/ip/ipidseq`
+1. Do: `set RHOSTS [ip]`
+1. Do: `run` or `exploit`
+
+## Options
+
+```
+Basic options:
+  Name       Current Setting  Required  Description
+  ----       ---------------  --------  -----------
+  INTERFACE                   no        The name of the interface
+  RHOSTS     10.0.20.254      yes       The target host(s)
+  RPORT      80               yes       The target port
+  SNAPLEN    65535            yes       The number of bytes to capture
+  THREADS    1                yes       The number of concurrent threads (max one per host)
+  TIMEOUT    500              yes       The reply read timeout in milliseconds
+```
+
+### Advanced Options
+```
```

Comment:
remove the code block whole section.  Add in (with the right formatting, see the example docs) GATEWAY_PROBE_HOST, SAMPLES, SECRET. i think those are module specific and can use some explanation

Reply:
Not quite sure what information i should add. the current explanation should be sufficient.
Thanks a lot.